deploys/demo-test.syncfor.science/nginx-ssl-proxy/docker-compose.yml

```diff
@@ -32,7 +32,7 @@ services:
       - certs:/etc/nginx/certs:ro
     entrypoint: /usr/local/bin/docker-gen -notify-sighup nginx_proxy -watch -only-exposed -wait 5s:30s /etc/docker-gen/templates/nginx.tmpl /etc/nginx/conf.d/default.conf
   lets-encrypt:
-    image: jrcs/letsencrypt-nginx-proxy-companion
+    image: alastaircoote/docker-letsencrypt-nginx-proxy-companion
```

Comment:
Oh my.

Yeah, between maintenance and the fact that this kind of setup will only work on a single-node deployment (where the companion and proxy can eavesdrop on just docker socket file), I'm much less enthusiastic about the surface-level simplicity than I was last March :)

Thanks for tracking this down!